src/pages/workspaces/workspace/analysis/modals/DeleteDiskChoices.ts

```diff
@@ -83,7 +106,10 @@ export const DeleteEnvironment = ({ id, existingEnvironmentConfig, deleteDiskSel
     }),
     div({ style: { lineHeight: '1.5rem' } }, [
       Utils.cond(
-        [runtime && persistentDisk && !runtime.persistentDiskAttached, () => {
+        [runtimeConfig && persistentDisk &&
+          (!isGceConfig(runtimeConfig) || isGceWithPdConfig(runtimeConfig)) && // this line checks if the runtime is a GCE VM with a PD attached
```

Comment:
<img width="1049" alt="Screenshot 2023-03-23 at 2 52 44 PM" src="https://user-images.githubusercontent.com/11773357/227318758-d8995107-2c95-4284-a4ce-1bc6e3e63652.png">

Attached is the current code coverage report, yellow means this part of the logic isn't checked.
Make sure there's a test case specifically for that case.